topics/contributing/slides/introduction.html

```diff
@@ -5,31 +5,119 @@
 logo: "GTN"
 ---
 
-### Training Resources
+### Some questions
 
-- Learn Galaxy @ Hub
+- How many of you are instructors, giving training/workshop?
```

Comment:
Are you instructor ...?
Similar to the others, "Do you use Galaxy for training?"